app/assets/stylesheets/components/_navbar.scss

```diff
@@ -1,3 +1,7 @@
+.btn-primary {
```

Comment:
btn-primary is a bootstrap class already. Please give this a unique name and then apply this class in the relevant views